thefiletree.docker

```diff
@@ -0,0 +1,26 @@
+# Take the latest version of nodejs container.
+FROM google/nodejs
+MAINTAINER David Aparicio "david.aparicio@free.fr"
+
+# Install tree build dependencies.
+RUN apt-get -qy install libicu-dev #node-gyp dependencies
```

Comment:
Nit: `libicu-dev` is effectively a Tree dependency because NPM sometimes seems to need it. No need for the comment at the end of the line. Also, it looks like `node-gyp` and `dependencies` are apt-get packages that you tried to install.